testcases/open_posix_testsuite/conformance/interfaces/pthread_cond_timedwait/2-7.c

```diff
@@ -51,7 +51,7 @@
 #define VERBOSE 1
 #endif
 
-#define TIMEOUT  (20)		/* Number of microseconds to wait */
+#define TIMEOUT_NS  (20000000)	/* Number of nanoseconds to wait */
```

Comment:
20us is 20000ns, so this is 1000x larger now.

Reply:
Maybe not a big deal for for the sleep, but this could overflow 32bit long on line 177:
```
ts.tv_nsec += TIMEOUT * 1000;
```

Reply:
You probably missed this one. TIMEOUT_NS is still 1000x larger. It should be "20000".

Reply:
Fixed -- thank you for spotting this!